src/data/reducers/assets.js

```diff
@@ -125,7 +125,7 @@ export const status = (state = {}, action) => {
       };
     case assetActions.UPLOAD_ASSET_FAILURE:
       return {
-        file: action.file,
+        asset: action.asset,
```

Comment:
I'm just changing the UPLOAD_ASSET_FAILURE reducer to reflect the changes I made to the uploadAssetFailure action creator, which renames the file key to asset.